app/components/manage-intermissions/edit.js

```diff
@@ -79,4 +83,49 @@ export default class manageIntermissionsEditComponent extends Component {
     yield intermission.destroyRecord();
     this.args.onClose();
   }
+
+  //position stuff
+  @tracked locationOptions = [
+    { code: 'during', name: 'Tijdens agendapunt' },
+    { code: 'before', name: 'Voor agendapunt' },
+    { code: 'after', name: 'Na agendapunt' }
+  ];
+
+  @tracked locationToBeSet;
+
+  get selectedLocation() {
+    if (this.locationToBeSet) {
+      return this.locationToBeSet;
+    } else {
+      return this.locationOptions.find(e => e.code == this.args.intermissionToEdit.position);
+    }
+  }
+  set selectedLocation(value) {
+    this.locationToBeSet = value;
+    if (this.selectedAp.content) {
+      this.args.intermissionToEdit.position = this.selectedLocation.code;
+    }
+    if (!value) {
+      this.locationToBeSet = null;
+      this.args.intermissionToEdit.position = null;
+      this.selectedAp = null;
+    }
+  }
+
+  get selectedAp() {
+    return this.args.intermissionToEdit.onderwerp;
+  }
+  set selectedAp(value) {
+    this.args.intermissionToEdit.position = this.selectedLocation.code;
```

Comment:
as mentioned in the backend PR (https://github.com/lblod/app-gelinkt-notuleren/pull/29 ) it might make sense to capture these in a separate model "DocumentPosition" or something similar.